packages/eslint-plugin-template/src/rules/no-any.ts

```diff
@@ -14,13 +14,15 @@ export default createESLintRule<Options, MessageIds>({
   meta: {
     type: 'suggestion',
     docs: {
-      description: `The use of "${ANY_TYPE_CAST_FUNCTION_NAME}" nullifies the compile-time benefits of the Angular's type system.`,
+      description: `The use of "${ANY_TYPE_CAST_FUNCTION_NAME}" nullifies the compile-time benefits of the Angular's type system`,
```

Comment:
I know you didn't make this change here, but makes sense to fix it while we are touching the message

```suggestion
      description: `The use of "${ANY_TYPE_CAST_FUNCTION_NAME}" nullifies the compile-time benefits of Angular's type system`,
```

Reply:
@rafaelss95 You marked this as resolved but you didn't make the change? 😅

Reply:
Ups, sorry, completely forgot this 😄